integration-cli/docker_cli_run_test.go

```diff
@@ -4549,7 +4550,7 @@ func (s *DockerSuite) TestRunAddDeviceCgroupRule(c *check.C) {
 
 // Verifies that running as local system is operating correctly on Windows
 func (s *DockerSuite) TestWindowsRunAsSystem(c *check.C) {
-	testRequires(c, DaemonIsWindowsAtLeastBuild(15000))
```

Comment:
This is somewhere between RS1 and RS2, so I took RS3

Reply:
Shouldn't it be RS2?